src/gui/EffectView.cpp

```diff
@@ -208,6 +214,43 @@ void EffectView::contextMenuEvent( QContextMenuEvent * )
 
 
 
+void EffectView::mousePressEvent(QMouseEvent* event)
+{
+	if (event->button() == Qt::LeftButton)
+	{
+		m_dragging = true;
+		m_opacityEffect->setOpacity(0.3);
+		QCursor c( Qt::SizeVerCursor );
+		QApplication::setOverrideCursor( c );
+		update();
+	}
+}
+
+void EffectView::mouseReleaseEvent(QMouseEvent* event)
+{
+	if (event->button() == Qt::LeftButton)
+	{
+		m_dragging = false;
+		m_opacityEffect->setOpacity(1);
+		QApplication::restoreOverrideCursor();
+		update();
+	}
+}
+
+void EffectView::mouseMoveEvent(QMouseEvent* event)
+{
+	if (!m_dragging) {return;}
```

Comment:
```suggestion
	if (!m_dragging) { return; }
```
Might be better to have spaces for padding in the block